python/delta_sharing/tests/test_reader.py

```diff
@@ -237,3 +241,164 @@ def list_files_in_table(
     expected = reader.to_pandas().iloc[0:0]
 
     pd.testing.assert_frame_equal(pdf, expected)
+
+
+def test_table_changes_to_pandas_non_partitioned(tmp_path):
+    # Create basic data frame.
+    pdf1 = pd.DataFrame({"a": [1, 2, 3], "b": ["a", "b", "c"]})
+    pdf2 = pd.DataFrame({"a": [4, 5, 6], "b": ["d", "e", "f"]})
+    pdf3 = pd.DataFrame({"a": [7, 8, 9], "b": ["x", "y", "z"]})
+    pdf4 = pd.DataFrame({"a": [7, 8, 9], "b": ["x", "y", "z"]})
+
+    # Add change type (which is present in the parquet files).
+    pdf1["_change_type"] = "Insert"
```

Comment:
For `insert/remove`, they are not from parquet files, and nit: first letter is not capital